src/stactools/spot/__init__.py

```diff
@@ -4,10 +4,11 @@
 
 
 def register_plugin(registry):
-    # Register subcommands
-
+    """Register subcommands"""
     from stactools.spot import commands
 
     registry.register_subcommand(commands.create_spot_command)
+
+
 __version__ = '0.1.5'
```

Comment:
The version policies for stactools packages are not well defined, but IMO since this will be an initial release of a new package, the version should either be `0.1.0` or `1.0.0`. I chose `0.1.0` for my [gap package](https://github.com/stactools-packages/gap/blob/7075f74cfa1e1c039ff18c34050cc51dea276c61/src/stactools/gap/__init__.py#L6) because I expect to make a bunch of changes to the package in the near future.